overlord/snapstate/refresh.go

```diff
@@ -38,7 +38,9 @@ import (
 // pidsOfSnap is a mockable version of PidsOfSnap
 var pidsOfSnap = cgroup.PidsOfSnap
 
-var genericRefreshCheck = func(info *snap.Info, canAppRunDuringRefresh func(app *snap.AppInfo) bool) error {
+// refreshCheck returns an error if the snap has processes running that aren't
+// services and aren't marked to be ignored (refresh-mode: "ignore-running").
+var refreshCheck = func(info *snap.Info) error {
```

Comment:
given that this is now explicitly about apps, it should be called maybe refreshAppsCheck ?